README.md

```diff
@@ -1,89 +1,74 @@
-```
+## Getting Started
 
-That's it if using a recent react-native version with [autolinking][autolinking] support.
+Get started with our 📝 [documentation](https://plaid.com/docs/link/react-native/) and the 📱[example project](https://github.com/plaid/react-native-plaid-link-sdk/blob/master/example/README.md), or ↔️ [Tiny Quickstart (React Native)](https://github.com/plaid/tiny-quickstart/tree/main/react_native) which is a end to end example demonstrating a minimal integration with this SDK.
 
-### Manual Integration
+If you're unfamiliar with React Native we recommend starting with the [environment setup instructions](https://reactnative.dev/docs/environment-setup).
 
-If using a version of react-native without [autolinking][autolinking]  support, then you will need to:
+In your React Native project directory:
 
 ```sh
-react-native link react-native-plaid-link-sdk
+npm install --save react-native-plaid-link-sdk
 ```
 
-followed by
+### iOS Setup
 
-1. In Xcode, in the project navigator, right click `Libraries` ▶ `Add Files to [your project's name]`
-2. Go to `node_modules` ▶ `react-native-plaid-link-sdk` ▶ `ios` and add `RNLinksdk.xcodeproj`
-3. In Xcode, in the project navigator, select your project. Add `libRNLinksdk.a` to your project's `Build Phases` ▶ `Link Binary With Libraries`
-4. Run your project (`Cmd+R`)<
+Add `Plaid` to your project’s Podfile as follows (likely located at `ios/Podfile`). The latest version is ![version](https://img.shields.io/cocoapods/v/Plaid).
 
-### OAuth Requirements
+Autolinking should install the CocoaPods dependencies for iOS project. If it fails you can run 
+
+```sh
+cd ios && bundle install && bundle exec pod install
+```
+
+#### OAuth Requirements
 
 :warning: All integrations must migrate to version 9.0.0 or later of the React Native SDK (requires version 4.1.0 or later of the iOS LinkKit SDK) by September 30, 2023, to maintain support for Chase OAuth on iOS. 
 
 For iOS OAuth to work, specific requirements must be met.
-* Redirect URIs must be registered, and set up as universal links ([docs](https://plaid.com/docs/link/ios/#register-your-redirect-uri))
+* Redirect URIs must be [registered](https://plaid.com/docs/link/ios/#register-your-redirect-uri), and set up as [universal links](https://developer.apple.com/documentation/xcode/supporting-associated-domains).
+
 
+### Android Setup
 
-## Android setup
-[Autolinking][autolinking]  unfortunately isn't supported for Android, because the SDK makes use of [TurboModules][turbomodules] to speed up app cold starts. Have a look at [#306](https://github.com/plaid/react-native-plaid-link-sdk/pull/306) for more context.
+- Android 5.0 (API level 21) and above.
+  - Your `compileSdkVersion` must be `33`.
+- Android gradle plugin `4.x` and above.
 
-### 1. Register your app id
-1. Log into your [Plaid Dashboard][plaid_dashboard] at the API page
+AutoLinking should handle all of the Android setup. 
+
+#### OAuth Requirements
+
+##### Register your app id
```

Comment:
nit: this should be "Register your app package name" if we keep this section structured like this

Reply:
I was wondering why that said app id 😂